wolfcrypt/src/asn.c

```diff
@@ -2933,6 +2933,8 @@ int UnTraditionalEnc(byte* key, word32 keySz, byte* out, word32* outSz,
     (void)rng;
 
     return totalSz + sz;
+    
+    (void)rng;
```

Comment:
This same (void)rng; cast is also above.  This one is not needed, please remove.